### ISSUE TITLE
test failure

### ISSUE THREAD

**Issue by hros** [FluxML/ONNX.jl#27]
running self tests ]test ONNX, results in a single error:
LoadError: LoadError: UndefVarError: CrossCor not defined
The error has been reproduced on several computers with different O/S all running Julia version 1.1.1

**Comment by amckay1** [comment]
Had similar issue, "CrossCor not defined" error when loading https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/duc on OSX and Julia 1.1.1.

Checking out Flux's master branch `]add Flux#master` fixed it for me, guessing will be resolved when master is merged into Flux's next stable release?

**Comment by ayush-1506** [comment]
Yes, looks like we'd need to wait for a new tagged version of Flux for this to go away(CrossCor was added just recently)
